src/site/_includes/partials/site-footer.njk

```diff
@@ -1,73 +1,83 @@
+{% from 'macros/icon.njk' import icon, svg with context %}
+
 <footer class="site-footer" role="contentinfo">
   <nav class="site-footer__primary-nav auto-grid" aria-label="footer navigation">
-    <div>
-      <h3 class="text-size-2 color-mid-text">Contribute</h3>
-      <ul class="w-footer__linkbox-list" role="list">
-        <li>
-          <a href="https://github.com/GoogleChrome/web.dev/issues/new?assignees=&labels=bug&template=bug_report.md&title="
-            data-category="Site-Wide Custom Events" data-label="Footer Link (index 1)">
-            {{ 'i18n.footer.file_a_bug' | i18n(locale) }}
-          </a>
-        </li>
-        <li>
-          <a href="https://github.com/googlechrome/web.dev"
-            data-category="Site-Wide Custom Events" data-label="Footer Link (index 2)">
-            {{ 'i18n.footer.view_source' | i18n(locale) }}
-          </a>
-        </li>
-      </ul>
+    <div class="site-footer__primary-nav-content">
+      <a href="/" class="site-footer__brand brand">
+        {{ svg('../../images/logo.svg', {label: 'web.dev'}) }}
+      </a>
+      <p class="text-size-0">We want to help you build beautiful, accessible, fast, and secure websites that work cross-browser, and for all of your users. This site is our home for content to help you on that journey, written by members of the Chrome team, and external experts.</p>
```

Comment:
This needs to go through i18n, like for example `{{ 'i18n.footer.mission' | i18n(locale) }}`

Reply:
I have moved all footer changes to another PR: https://github.com/GoogleChrome/web.dev/pull/9520